build/presubmit.sh

```diff
@@ -19,6 +19,4 @@ set -x
 
 ./build/check_gofmt.sh .
 ./build/check_boilerplate.sh
-go vet github.com/google/cadvisor/...
-godep go test -v -race -test.short github.com/google/cadvisor/...
-godep go build github.com/google/cadvisor
+make
```

Comment:
how about instead of putting this here, we add a presubmit make rule that depends on the necessary other rules? (then change the presubmit call in the jenkins script to call make instead)